package.json

```diff
@@ -16,10 +16,13 @@
     },
     "main": "main.js",
     "devDependencies": {
+        "concurrently": "7.0.0",
```

Comment:
I hope these 3 dependencies are acceptable because they make things more reliable (starting server + app + cypress without `concurrently` is hard, and starting `cypress run` before making sure the app and server are ready -- `wait-on` -- can lead to timeouts).